fastify.js

```diff
@@ -224,6 +260,8 @@ function build (options) {
     }
 
     req.id = genReqId(req)
+    handleTrustProxy(req)
+    req.hostname = req.hostname || req.headers['host']
```

Comment:
I think we should add `ip` and `hostname` directly to our request and reply object.
Maybe we can do that in the `next` branch.

Reply:
Yes, definitely.